lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -107,6 +114,13 @@
     repo: 'ftp://download.nvidia.com/opensuse/12.2'
     state: absent
 
+# Add NVIDIA repository for graphics drivers with additional arguments
```

Comment:
We recommend Ansible users to always add a `name:` to a task. So we should do the same for the examples.
```suggestion
- name: Add NVIDIA repository for graphics drivers with additional arguments
  zypper_repository:
```
Please also improve the other examples.

Reply:
Done